ci-operator/step-registry/assisted/installer/multi-arch/assisted-installer-multi-arch-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
```

Comment:
I'd like to make this workflow (and steps) more generalized, as it doesn't have anything specific with assisted-installer component
For that we'd want to do the following changes:
* Have the workflow under ``step-registry/assisted/tools/``
* The steps themselves can use ``from: src`` so that they have all relevant Dockerfiles
* Step parameters are needed to know which Dockerfiles to build and what name should be chosen for their images

Reply:
I am not sure how can it be done:
The workflow references **assisted-installer-multi-arch-setup** and **assisted-installer-multi-arch-build** which are repo specific.

Reply:
Part of it is making them not repo-specific, and moving them along